frontend/package-lock.json

```diff
@@ -2595,262 +2595,6 @@
       "resolved": "https://registry.npmjs.org/@emotion/weak-memoize/-/weak-memoize-0.3.1.tgz",
       "integrity": "sha512-EsBwpc7hBUJWAsNPBmJy4hxWx12v6bshQsldrVmjxJoc3isbxhOrF2IcCpaXxfvq03NwkI7sbsOLXbYuqF/8Ww=="
     },
-    "node_modules/@esbuild/aix-ppc64": {
```

Comment:
@rawagner @jgyselov Not sure why all these optional @esbuild packages were removed, but this caused an issue in our downstream build system, which does a multi-platform build. Can you please check your `npm` version and settings to see if something caused these to be removed... or perhaps it was a merge error.